test/integration/helpers.js

```diff
@@ -282,7 +287,7 @@ function resolveFixturePath(fixture) {
   if (path.extname(fixture) !== '.js') {
     fixture += '.fixture.js';
   }
-  return path.join('test', 'integration', 'fixtures', fixture);
+  return path.resolve('test', 'integration', 'fixtures', fixture);
```

Comment:
does this actually change anything?

Reply:
This was necessary before but I changed the signature of `runMochaJSONRawAsync` to not use a special `fixturePath` argument. This allows us to revert this change and make the code more understandable.

https://github.com/mochajs/mocha/pull/3915/commits/3f6372e1b138756edaf89cea2a578ec8cb6a1a5a